### DISCUSSION TITLE
Robyn Calibration : Liftabs is pretty low as compared to the decomposition in Robyn output

### DISCUSSION THREAD

**Discussion by rkal1809** [facebookexperimental/Robyn#816] [Q&A]
Hello, I am building MMM using Robyn package. I am using its calibration feature for one of the organic variables, to have results aligned with previous experinments. The sum total of Liftabs is 4K(in robyn code as input), which is spread across last 4years. In the decomposition file of robyn output, the xDecompAgg ranges from 5.5K to 8K for different model solID. I expected the decomposition of this variable to center around 4K only. Can someone please help me understand what is the possible reason for this extrapolated contribution of this variable and how is calibration piece working here?

**Comment by gufengzhou** [comment]
Hi, you have a study that went across 4 years? that sounds not very realistic. otherwise, Robyn requires more iterations when calibrating, so I suggest you increase the iterations significantly and retry.

**Reply by rkal1809** [reply]
Thanks for reverting. Even if I reduce the time period for robyn input, I am facing this issue. I am running 5K iterations with 10trials. Do I need to increase it further? Also, is it possible that lift specified in calibration off from decomposition data in robyn output?

**Reply by gufengzhou** [reply]
usually a study lasts between 1-4 weeks. is it something you're using? i can imagine there might be edge cases where robyn can't find a good fit when calibrating. but i'd need more info to make an informed impression. you might increase iterations, because more iteration means better convergence, while trials are for different starting points / local optima. you could decrease trials as a trade off.
